### ISSUE TITLE
[morty] python-*: do_install() failed

### ISSUE THREAD

**Issue by zuka0828** [meta-debian/meta-debian#283]
The same error happens while building `python-openssl`, `python-ply`, and `python-pycparser`.

Build conditions & Extra settings in local.conf: Same as #281

Error:

```
ERROR: python setup.py install execution failed.
running install
error: invalid Python installation: unable to open /home/deby/poky/build-world/tmp/sysroots/qemuarm/usr/lib/python2.7/config-arm-linux/Makefile (No such file or directory)
WARNING: exit code 1 from a shell command.
```

**Comment by hieunv-tsdv** [comment]
I updated the _debian-multiarch.bbclass_ to fix the problem in python-* recipes.

*do_install* failed if the recipe inherits "allarch" class.

please check the commit:  b466cca

**Comment by zuka0828** [comment]
Fixed by #286, thanks a lot ! @hieunv-tsdv
